robot/impl/local_robot.go

```diff
@@ -767,9 +767,13 @@ func (r *localRobot) updateDefaultServices(ctx context.Context) {
 	}
 }
 
-// Refresh does nothing for now.
+// Refresh causes the web service to reload it's subtype service maps from the actual robot resources.
 func (r *localRobot) Refresh(ctx context.Context) error {
```

Comment:
Should I not be using this function? It felt like it fit in the paradigm, and my thinking was if MORE stuff needs to be refreshed here later, it can be added.

Reply:
I think this makes sense here